mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  return `£${calculateSalesTax(price).toFixed(2)}`;
```

Comment:
For this one, I would suggest declare a new variable to store the value in advance and return that variable, to make the code clearer, e.g. formatCurrency = calculateSalesTax(price).toFixed(2) and then return formatCurrency.